unskript-ctl/templates/timeout_handler.j2

```diff
@@ -0,0 +1,30 @@
+import threading
+import functools
+
+
+def timeout(seconds=10, error_message="Function call timed out"):
```

Comment:
`seconds` should be same as the default value for check timeout. I think its 60s?